packages/charts/react-charting/src/components/AreaChart/AreaChart.base.tsx

```diff
@@ -119,9 +126,13 @@ export class AreaChartBase extends React.Component<IAreaChartProps, IAreaChartSt
   private _enableComputationOptimization: boolean;
   private _firstRenderOptimization: boolean;
   private _emptyChartId: string;
+  private _cartesianChartRef: React.RefObject<IChart>;
 
   public constructor(props: IAreaChartProps) {
     super(props);
+
+    initializeComponentRef(this);
```

Comment:
>initializeComponentRef [](http://example.com/codeflow?start=4&length=22)

is this needed for controls using cartesian chart as this is already getting initialized in cartesian chart

Reply:
Yes, this is required to access the public members of AreaChart from outside. The issue is that we are not extending AreaChart from the CartesianChart component but using it as a child component, so the same ref can't be used.

Reply:
but the chartContainerDiv is inside Cartesian only. Can we hold the ref of cartesian and pass outside as needed.

Reply:
Yes. Pushed the change.